.github/CODEOWNERS

```diff
@@ -2194,6 +2202,21 @@ x-pack/test/security_solution_cypress/cypress/tasks/expandable_flyout  @elastic/
 /x-pack/plugins/security_solution/public/resolver @elastic/security-threat-hunting-investigations
 /x-pack/plugins/security_solution/public/threat_intelligence @elastic/security-threat-hunting-investigations
 /x-pack/plugins/security_solution/public/timelines @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/common/components/header_actions @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/common/types/header_actions @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/flyout/network_details @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/flyout/rule_details @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/investigations @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/detections/configurations/security_solution_detections @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/detections/pages/detection_engine/detection_engine.tsx @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/common/hooks/use_resolve_conflict.tsx @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/common/components/drag_and_drop @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/common/components/draggables @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/common/components/events_tab @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution_serverless/public/upselling/pages/threat_intelligence_paywall.tsx @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/common/mock/mock_timeline_control_columns.tsx @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/common/components/exit_full_screen @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/app/home/template_wrapper/timeline/index.tsx @elastic/security-threat-hunting-investigations
```

Comment:
we could the `x-pack/plugins/security_solution/public/app/home/template_wrapper/timeline` folder here instead of the single `index.tsx` file within?

Reply:
Yep, will update!

Reply:
done!